src/Batch/Client/Src/GeneratedProtocol/AccountOperations.cs

```diff
@@ -1,58 +1,33 @@
-// Copyright (c) Microsoft and contributors.  All rights reserved.
```

Comment:
@matthchr was the license header removed from these files intentionally? If not, @tbombach has anything changed to cause this?